app/views/managers/my_submitted_forms.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+<table class="table table-bordered mt-4 form-table">
+  <thead>
+  <tr>
+    <% @submissions.first.data.keys.each do |key| %>
+      <th scope="col"> <%= key.titleize%> </th>
+    <% end %>
+    <th scope="col"> Status </th>
+  </tr>
+  </thead>
```

Comment:
need to change name of the view my_submitted_form is not a good naming convention